source/gameengine/Ketsji/KX_KetsjiEngine.cpp

```diff
@@ -898,10 +925,6 @@ void KX_KetsjiEngine::RenderFrame(KX_Scene *scene, KX_Camera *cam, unsigned shor
 
 	KX_SetActiveScene(scene);
 
-#ifdef WITH_PYTHON
```

Comment:
This has to be restored.... I don't know how... When I rebase this leads to weird things. This is why I prefer to make new branches hehe